src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomColor = new Random();
+
+    public Color getRandomColor() {
+        return Color.values()[randomColor.nextInt(Color.values().length)];

```

Comment:
Better to introduce `randomColor.nextInt(Color.values().length)` as local variable